modules/workflow-workflowoperation/src/main/java/org/opencastproject/workflow/handler/workflow/TagWorkflowOperationHandler.java

```diff
@@ -119,6 +119,20 @@ public WorkflowOperationResult start(WorkflowInstance workflowInstance, JobConte
       elementSelector.addTag(tag);
     }
 
+    boolean wildcardInTargetFlavorType = false;
+    boolean wildcardInTargetFlavorSubtype = false;
+    if (configuredTargetFlavor != null) {
+      String[] targetFlavorTypes = configuredTargetFlavor.split("/");
+      if (targetFlavorTypes.length != 2) {
+        String errorMsg = String.format("The configured target flavor has a wrong format. "
+                + "The configured target flavor: '%s'.", configuredTargetFlavor);
+        logger.error(errorMsg);
+        throw new RuntimeException(errorMsg);
```

Comment:
Don't log an error when you throw an exception anyway. The result will just be that you see the same error twice in the logs. Also, I like short messages ;-)

```suggestion
        final String errorMsg = String.format("The configured target flavor `%s` has a wrong format.", configuredTargetFlavor);
        throw new RuntimeException(errorMsg);
```